com.spotify.Client.json

```diff
@@ -310,7 +310,8 @@
                 "--enable-decoder=mp3",
                 "--enable-decoder=mp3adu",
                 "--enable-decoder=opus",
-                "--enable-decoder=vorbis"
+                "--enable-decoder=vorbis",
+                "--disable-inline-asm"
```

Comment:
This has significant perf impact. The proper fix is to [patch](https://github.com/flathub/io.qt.qtwebengine.BaseApp/blob/branch/5.15-23.08/patches/chromium-ffmpeg-binutils.patch) it .

Reply:
Thanks for the tip. I'll include the patch and remove the option